examples/webgl_loader_collada_kinematics.html

```diff
@@ -120,14 +120,10 @@
 
 				// Lights
 
-				var directionalLight = new THREE.HemisphereLight( 0xffeeee, 0x111122 );
-				directionalLight.position.x = Math.random() - 0.5;
```

Comment:
The default is `0, 100, 0` though. Wouldn't it be better to make the default `1, 1, 1`?